app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -642,15 +641,13 @@ export function testsForRpcMethodSupportingBlockParam(
     ['given no block tag', 'none'],
```

Comment:
The reason for the two functions was to test that if the block param was not passed then it would default to latest. That doesn't appear to happen now — we always pass a block param.

Reply:
It's set to `undefined` in all of the cases where the "without block param" function was used. So it should be doing the same thing.

Reply:
Ah. What do you think about making that explicit by adding `undefined` as the third value in this array?

Reply:
Sure, we can do that

Reply:
Done in https://github.com/MetaMask/metamask-extension/pull/16856/commits/9511b719887835ae24efc32698a542bbe6538af1